tests/test_api.py

```diff
@@ -500,18 +500,6 @@ def get_combined_filenames() -> set[str]:
             filenames = {relative_filename(f).replace("\\", "/") for f in data.measured_files()}
             return filenames
 
-        # Case 1: get the order right.
-        make_files()
-        self.make_file(".coveragerc", """\
-            [paths]
-            plugins =
-                plugins/
-                ci/girder/plugins/
```

Comment:
This is weird; I didn't touch this, and these files remain on my local repository. Go figure, I'll check it out later.